ruby/ql/lib/codeql/ruby/ApiGraphs.qll

```diff
@@ -271,6 +271,26 @@ module API {
     DataFlow::CallNode getCallNode() { this = Impl::MkMethodAccessNode(result) }
   }
 
+  /**
+   * An API entry point.
```

Comment:
What is an entry point? Could we add an example or a bit of explanation here?

Reply:
EntryPoints are used when we want `API::Node`s for something that is not connected to the root of the API-graph.   
In JS that is usually references to global variables for specific library models (including all global variables in the API-graph would blow up).    
[Here is an example of how we've used an EntryPoint](https://github.com/github/codeql/blob/bcdbfefb2b49b5aa6d483fdb703097780056b070/javascript/ql/lib/semmle/javascript/frameworks/History.qll#L7-L21).

Reply:
Elaborated a bit